src/EditorFeatures/Core/EditorFeatures.csproj

```diff
@@ -55,19 +55,27 @@
       <Private>false</Private>
     </Reference>
     <Reference Include="Microsoft.VisualStudio.ImageCatalog, Version=$(VisualStudioReferenceAssemblyVersion), Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
-    <Reference Include="Microsoft.VisualStudio.Imaging, Version=$(VisualStudioReferenceAssemblyVersion), Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
-    <Reference Include="Microsoft.VisualStudio.Imaging.Interop.14.0.DesignTime, Version=$(VisualStudioReferenceAssemblyVersion), Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
+    <Reference Include="Microsoft.VisualStudio.Imaging, Version=$(VisualStudioReferenceAssemblyVersion), Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
+      <HintPath>C:\Program Files (x86)\Microsoft Visual Studio 15.0\Common7\IDE\Microsoft.VisualStudio.Imaging.dll</HintPath>
```

Comment:
It would be better to use a property/environment variable for the hint paths, especially since C: isn't always the install drive.  On my box `%DevEnvDir%` (and `$(DevEnvDir)`) resolve to `C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE`.

(this is all assuming that `%DevEnvDir%` is set/updated during Dev15 installation.)